packages/contracts-bedrock/src/L1/L1CrossDomainMessenger.sol

```diff
@@ -20,24 +21,45 @@ contract L1CrossDomainMessenger is CrossDomainMessenger, ISemver {
     /// @custom:network-specific
     OptimismPortal public portal;
 
+    /// @notice Address of the SystemConfig contract.
+    SystemConfig public systemConfig;
+
     /// @notice Semantic version.
-    /// @custom:semver 2.3.0
-    string public constant version = "2.3.0";
+    /// @custom:semver 2.4.0
+    string public constant version = "2.4.0";
 
     /// @notice Constructs the L1CrossDomainMessenger contract.
     constructor() CrossDomainMessenger() {
-        initialize({ _superchainConfig: SuperchainConfig(address(0)), _portal: OptimismPortal(payable(address(0))) });
+        initialize({
```

Comment:
I believe I mentioned this in a prior review, but would prefer `_disableInitializers` everywhere since its safer (and cleaner/more semantically correct)—some vars, like the `Ownable` owner, should not be initialized to the zero address, so `_disableInitializers` mitigates the risk of initializing to a bad value

Reply:
Oh interesting, I didn't realize that would affect the tests. Ok, we can follow up with that